### ISSUE TITLE
Add skip to search link in a11y-nav

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#5919]
### Summary

We used to have a "skip to search" link as part of the a11y-nav. I believe it was temporarily removed. We should add the link again now that I think the affected pieces are stable.

### URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

### Reproduction steps

Open the above page and press the tab key. Links to the main content and language selector are available. It is missing a link to search.

### Expected behavior

a11y-nav should also contain a link to skip to search.

### Actual behavior

See "Reproduction steps"

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Mac OS

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
